lib/brute/http_basic_auth/engine.py

```diff
@@ -32,7 +45,7 @@ def extra_requirements_dict():
 
 
 def login(user, passwd, target, port, timeout_sec, log_in_file, language, retries, time_sleep, thread_tmp_filename,
-          socks_proxy, scan_id, scan_cmd):
+          socks_proxy, scan_id, scan_cmd, headers):
```

Comment:
please do not pass headers as parameters as this is breaking the core framework - headers are not parameters which are passed from the core